src/hapi/pipelines/database/locations.py

```diff
@@ -1,3 +1,5 @@
+"""Populate the location table."""
```

Comment:
I feel like our linter should be catching these missing docstrings...

Reply:
That would be nice. I'm not sure if there's a way to configure it to do that. I've created a ticket so we don't forget: https://humanitarian.atlassian.net/browse/HDXDSYS-782